hwilib/cli.py

```diff
@@ -117,14 +118,12 @@ def error(self, message):
         self.exit(2)
 
 def process_commands(cli_args):
-    CHAINS = ['main', 'test', 'regtest', 'signet']
-
     parser = HWIArgumentParser(description='Hardware Wallet Interface, version {}.\nAccess and send commands to a hardware wallet device. Responses are in JSON format.'.format(__version__))
     parser.add_argument('--device-path', '-d', help='Specify the device path of the device to connect to')
     parser.add_argument('--device-type', '-t', help='Specify the type of device that will be connected. If `--device-path` not given, the first device of this type enumerated is used.')
     parser.add_argument('--password', '-p', help='Device password if it has one (e.g. DigitalBitbox)', default='')
     parser.add_argument('--stdinpass', help='Enter the device password on the command line', action='store_true')
-    parser.add_argument('--chain', help='Select chain to work with ({})'.format(', '.join(CHAINS)), default='main', choices=CHAINS)
+    parser.add_argument('--chain', help='Select chain to work with ({})'.format(', '.join(Chain.names())), default='main', choices=Chain.names())
```

Comment:
It's unnecessary to list out the names again here. The `choices` option will do that automatically.

This could also do the same trick that `AddressType` does to do the automatic conversion from string to enum type without needing `Chain.by_name` below.

Reply:
Addressed in the comment below